packages/ember-htmlbars/lib/keywords/element-component.js

```diff
@@ -42,7 +44,7 @@ function getComponentPath(param, env) {
   return path;
 }
 
-function render(morph, env, scope, [path, ...params], hash, template, inverse, visitor) {
+function render(morph, env, scope, [path, ...params], hash, template, inverse, visitor, isRerender=false) {
```

Comment:
When is `isRerender` true? Nothing is passed in your changes to `keywords/component.js`?